bundle-ml/src/main/scala/ml/combust/bundle/dsl/Bundle.scala

```diff
@@ -104,7 +105,8 @@ object Bundle {
     apply(BundleInfo(uid = UUID.randomUUID(),
       name = name,
       format = format,
-      version = Bundle.version), root)
+      version = Bundle.version,
+      LocalDateTime.now().toString), root)
```

Comment:
Also, to be sure of the field that is being set, just do a quick:
timeCreated = LocalDateTime.now().toString